pallets/gear-eth-bridge/src/tests.rs

```diff
@@ -0,0 +1,642 @@
+            .collect::<Vec<u8>>();
+        let authority_set_hash: H256 = Blake2_256::hash(&authority_set_ids_concat).into();
+
+        run_to_block(67);
+        do_events_assertion(
+            11,
+            67,
+            [
+                SessionEvent::NewSession { session_index: 11 }.into(),
+                Event::AuthoritySetHashChanged(authority_set_hash).into(),
+            ],
+        );
+
+        assert_eq!(
+            AuthoritySetHash::get().expect("infallible"),
+            authority_set_hash
+        );
+        assert!(!QueueMerkleRoot::get().expect("infallible").is_zero());
+
+        run_to_block(73);
+        do_events_assertion(
+            12,
+            73,
+            [SessionEvent::NewSession { session_index: 12 }.into()],
+        );
+
+        on_finalize_gear_block(73);
+        System::assert_last_event(GrandpaEvent::NewAuthorities { authority_set }.into());
+
+        System::reset_events();
+
+        on_initialize(74);
+        do_events_assertion(12, 74, [Event::BridgeCleared.into()]);
+
+        assert!(!AuthoritySetHash::exists());
+        assert!(QueueMerkleRoot::get().expect("infallible").is_zero());
+
+        run_to_block(102);
+        do_events_assertion(
+            16,
+            102,
+            [
+                SessionEvent::NewSession { session_index: 13 }.into(),
+                SessionEvent::NewSession { session_index: 14 }.into(),
+                SessionEvent::NewSession { session_index: 15 }.into(),
+                SessionEvent::NewSession { session_index: 16 }.into(),
+            ],
+        );
+
+        let authority_set = era_validators_authority_set(18);
+        let authority_set_ids_concat = authority_set
+            .clone()
+            .into_iter()
+            .flat_map(|(public, _)| public.into_inner().0)
+            .collect::<Vec<u8>>();
+        let authority_set_hash: H256 = Blake2_256::hash(&authority_set_ids_concat).into();
+
+        run_to_block(103);
+        do_events_assertion(
+            17,
+            103,
+            [
+                SessionEvent::NewSession { session_index: 17 }.into(),
+                Event::AuthoritySetHashChanged(authority_set_hash).into(),
+            ],
+        );
+
+        run_to_block(109);
+        on_finalize_gear_block(109);
+        do_events_assertion(
+            18,
+            109,
+            [
+                SessionEvent::NewSession { session_index: 18 }.into(),
+                GrandpaEvent::NewAuthorities { authority_set }.into(),
+            ],
+        );
+
+        on_initialize(110);
+        do_events_assertion(18, 110, [Event::BridgeCleared.into()]);
+    })
+}
+
+#[test]
+fn bridge_is_not_yet_initialized_err() {
+    init_logger();
+    new_test_ext().execute_with(|| {
+        const ERR: Error = Error::BridgeIsNotYetInitialized;
+
+        run_to_block(1);
+        run_block_and_assert_bridge_error(ERR);
+
+        run_to_block(35);
```

Comment:
```suggestion
        run_to_block(6 * 6 - 1);
```
better with `const`s